viewing_party/party.py

```diff
@@ -1,23 +1,139 @@
 # ------------- WAVE 1 --------------------
 
 def create_movie(title, genre, rating):
-    pass
+    
+    if title and genre and rating:
+        return {
+        "title": title,
+        "genre": genre, 
+        "rating": rating
+        }
+    else:
+        return None
+        
+def add_to_watched(user_data, movie):
+    user_data["watched"].append(movie)
+    return user_data 
+
+def add_to_watchlist(user_data, movie):
+    user_data["watchlist"].append(movie)
+    return user_data
+
+def watch_movie(user_data, title):
+    # watchlist = user_data["watchlist"]
+    # watched = user_data["watched"]
+
+    for movie in user_data["watchlist"]:
+        if movie["title"] == title:
+            user_data["watched"].append(movie)
+            user_data["watchlist"].remove(movie)

```

Comment:
Consider `break`ing out of the looping after performing the remove/append logic. Modifying a loop while iterating over it (the watchlist list here) can result in unexpected behavior. It's probably not a problem here, since we only expect to find one of the movie in the list, but we can make that explicit by using `break` to terminate the iteration as soon as we modify the list.

Also, a blank line after this would help visually separate the `return` (which is outside the loop) from the loop body.